portfolio/src/main/webapp/script.js

```diff
@@ -12,70 +12,70 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Opens the selected content.
- */
 
+// Function that opens the selected content.
 function openContent(evt, page) {
-  let i, content, navitem;
+  let i;
 
-  content = document.getElementsByClassName("content");
+  // Set all the pages display to none
+  const content = document.getElementsByClassName("content");
   for (i = 0; i < content.length; i++) {
     content[i].style.display = "none";
   }
 
-  navitem = document.getElementsByClassName("navitem");
+  // Once a page is selected make it active and display it
+  const navitem = document.getElementsByClassName("navitem");
   for (i = 0; i < navitem.length; i++) {
     navitem[i].className = navitem[i].className.replace(" active", "");
   }
 
   document.getElementById(page).style.display = "block";
   evt.currentTarget.className += " active";
 
+  // Select the header and footer
+  const headerElement = document.getElementById("headerSection");
+  const footerElement = document.getElementById("footerSection");
+
+  // If on the about page make the header and footer large otherwise shrink them
   if(page == "about"){
-    let headerElement = document.getElementById("headerSection");
     headerElement.classList.remove("minimise");
     headerElement.classList.add("maximise");
-    let footerElement = document.getElementById("footerSection");
     footerElement.classList.add("maximiseFooter");
     footerElement.classList.remove("minimiseFooter");
   }else{
-    let headerElement = document.getElementById("headerSection");
     headerElement.classList.add("minimise");
     headerElement.classList.remove("maximise");
-    let footerElement = document.getElementById("footerSection");
     footerElement.classList.remove("maximiseFooter");
     footerElement.classList.add("minimiseFooter");
   }
+
+  // When comments page is selected load the comments
+  if (page == "comments"){
+    getComments();
+  }
 }
 
-/**
- * Fetches a my name from the server and adds it to the DOM.
- */
-function getName() {
-  // The fetch() function returns a Promise because the request is asynchronous.
-  const responsePromise = fetch('/data');
+// Function to fetch comments from /data
+function getComments() {
```

Comment:
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
Fixed